InnerEye/ML/reports/notebook_report.py

```diff
@@ -48,7 +53,7 @@ def generate_notebook(template_notebook: Path, notebook_params: Dict, result_not
             resources=resources,
             notebook_name=str(result_notebook.stem)
         )
-    return result_notebook.with_suffix(resources['output_extension'])
+    return resources
```

Comment:
does nto need to return resources at all. Rather, it should return result_notebook.with_suffix(resources['output_extension'])